OpenUpSA/wazimap-ng-ui
# switching to another version if the geography does not exist in the selected version

## Description
switching to another version in which the selected geography exists, if the geography does not exist in the selected version

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-508

## How is it tested automatically?
added test steps into `__tests__/gui/multi_boundary_versions.feature`

## How should a reviewer test it locally
* navigate to a geography which does not exist in the current(or default) version. e.g. #geo:87ad36151ffffff in sandbox.youthexplorer
* confirm that the page is loaded successfuly
      - it switched to another version automatically
      - data mapper and the other elements are not stuck in the loading state

## Screenshots


## Changelog

### Added

### Updated
* `src/js/versions/version_controller.js` to switch to another version

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
